Modix.Services/AutoRemoveMessage/AutoRemoveMessageSetup.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+using Modix.Services.Messages.Discord;
+using Modix.Services.Messages.Modix;
+
+namespace Modix.Services.AutoRemoveMessage
+{
+    /// <summary>
+    /// Contains extension methods for configuring the auto remove message feature upon application startup.
+    /// </summary>
+    public static class AutoRemoveMessageSetup
+    {
+        /// <summary>
+        /// Adds the services and classes that make up the auto remove message service to a service collection.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> to which the auto remove message services are to be added.</param>
+        /// <returns><paramref name="services"/></returns>
+        public static IServiceCollection AddAutoRemoveMessage(this IServiceCollection services)
+            => services
+                .AddScoped<IAutoRemoveMessageService, AutoRemoveMessageService>()
+                .AddScoped<INotificationHandler<RemovableMessageSent>, AutoRemoveMessageHandler>()
+                .AddScoped<INotificationHandler<ReactionAdded>, AutoRemoveMessageHandler>()
+                .AddScoped<INotificationHandler<RemovableMessageRemoved>, AutoRemoveMessageHandler>();
+    }
```

Comment:
Will these resolve to the same instance within the same scope, or separate instances? (more of a general question, could you try it out?)

Reply:
These would resolve to separate instances.